src/App.tsx

```diff
@@ -27,6 +28,22 @@ const HomeStack = () => {
       <Stack.Screen name="WelcomeScreen" component={WelcomeScreen} />
       <Stack.Screen name="LoginScreen" component={LoginScreen} />
       <Stack.Screen name="SignUpScreen" component={SignUpScreen} />
+      <Stack.Screen
```

Comment:
I added the `Password Changed Screen` to the `HomeStack` component and to the tab navigation below. Is this the proper way of implementing it? 

I implemented it this way because when I initially tried: `<Stack.Screen name="PasswordChanged" component={PasswordChanged} />`, the left arrow wouldn't appear on the Password Changed Screen.

Reply:
I believe that's right, it's how the UI manages navigation. But make sure it's "stacked" correctly on what it needs to go <- back to. It would be strange for it to appear and the back screen to not be the previously seen screen.